README.md

```diff
@@ -153,21 +165,64 @@ We have three main branches in the repository, `master`, `stage` and `dev`. Any
 
 **Dev** is deployed from the `dev` branch. The url for the deployment is https://dev.data.gieffektivt.no. This environment uses the development database, and is unstable. If you wish to test that your code runs in the google cloud environment, you may merge your changes into the dev branch.
 
-|Branch|Url|Database|Environment variables|
-|-|-|-|-|
-|`Master`|https://data.gieffektivt.no|`EffektDonasjonDB`|Production|
-|`Stage`|https://stage.data.gieffektivt.no|`EffektDonasjonDB`|Production|
-|`Dev`|https://dev.data.gieffektivt.no|`EffektDonasjonDB_Dev`|Development|
+| Branch   | Url                               | Database               | Environment variables |
+| -------- | --------------------------------- | ---------------------- | --------------------- |
+| `Master` | https://data.gieffektivt.no       | `EffektDonasjonDB`     | Production            |
+| `Stage`  | https://stage.data.gieffektivt.no | `EffektDonasjonDB`     | Production            |
+| `Dev`    | https://dev.data.gieffektivt.no   | `EffektDonasjonDB_Dev` | Development           |
 
 ### Google cloud build
 
-## Authentification
-
-## Code structure
+## Code Structure & Implementation Details
 
 ### Routes
 
+Routes in `/routes` define which API endpoints exist, and what they do. They're roughly broken into files (and corresponding url sub-paths) by grouped functionality.
+
+Routes are loaded into the app in `server.js`, and that set of endpoints is assigned a url subpath:
+```
+app.use('/donors', donorsRoute)
+```
+By convention, each subpath will have its own file in `/routes` with the same name, for example url `/donors` is implemented in `/routes/donors.js`
+
+At a high level, here's what the subpaths do:
+- `/auth`: authentication - logging in/out, passwords, and tokens
+- `/donors`: CRUD endpoints for donor objects
+- `/donations`: CRUD endpoints for donation objects
+- `/distributions`: CRUD endpoints for donation percent allocations
+- `/organizations`: CRUD endpoints for organization objects (example: Against Malaria Foundation)
+- `/payment`: CRUD endpoints for metadata (example: fees) about payment methods
+- `/referrals`: CRUD endpoints for donor referrals
+- `/paypal`: endpoints for Paypal payments
+- `/vipps`: endpoints for Vipps payments and payment agreements
+- `/avtalegiro`: endpoints for Avtalegiro payments and payment agreements
+- `/facebook`: endpoints for Facebook payments
+- `/scheduled`: endpoints called by cron jobs (usually daily), mostly for processing recurring payments
+- `/logging`: When we process recurring payments, we store some info. These endpoints fetch that info.
+- `/mail`: endpoints for sending certain emails
+- `/reports`: manual import data in bulk, via documents from banks or payment providers
+- `/csr`: ?
```

Comment:
couldn't figure out what this was an acronym for, or what the grouped functionality was here

Reply:
@UberFarmer CSR is an unused endpoint which previously was used from som Corporate Social Responsibility stuff. We should probably just removed all of that code :)